node/src/main/scala/coop/rchain/node/revvaultexport/mainnet1/StateBalanceMain.scala

```diff
@@ -56,11 +56,6 @@ object StateBalanceMain {
 
   // hard-coded value in RevVault.rho
   val genesisVaultMapDepth = 2
-  // The way to get this Unforgeable name needs reporting casper to get all the concrete comm events.
-  // Anyway, as long as RevVault.rho and genesis doesn't change, this value would be fixed.
-  val genesisVaultMapPar: Par = GPrivate(
-    "5e8200d74ba8689013cf86f075775a18a42f1bd3a5ea122e82dfca9ba485a924".unsafeHexToByteString
-  )
```

Comment:
The `vaultMap` is no longer a fixed unforgeable name because of the changes we made. The future testnet and mainnet could be totally different. It depends on the genesis.